tx-pool/src/component/pending.rs

```diff
@@ -262,8 +288,33 @@ mod tests {
             .sorted_keys()
             .map(|key| key.id.clone())
             .collect::<Vec<_>>();
-        // the entry with most ancestors score will win
-        let expect_result = tx2_4.proposal_short_id();
-        assert_eq!(txs_sorted_by_fee_rate[0], expect_result);
+
+        let expect_result = vec![
```

Comment:
Please put the test to a standalone test case, these two functions serve for differrence purposes, and should be two test cases.

Reply:
I think it's better to put them together, becasue `keys_sorted_by_fee_and_relation` is based on `keys_sorted_by_fee`, it's easy for us to compare the different outputs of them.